src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -116,7 +138,7 @@
           DependsOnTargets="DiscoverTestInputs;CheckTestCategories"
           AfterTargets="CheckTestCategories"
           Inputs="@(RunTestsForProjectInputs)"
-          Outputs="$(TestsSuccessfulSemaphore);$(TestPath)%(TestTargetFramework.Folder)/$(XunitResultsFileName);$(CoverageOutputFilePath)"
+          Outputs="$(TestsSuccessfulSemaphore);$(TestPath)%(TestNugetTargetMoniker.Folder)/$(XunitResultsFileName);$(CoverageOutputFilePath)"
```

Comment:
Sure I don't think we need to do it as part of this but it is clean-up that we need to do.